### DISCUSSION TITLE
Let's talk about OCR

### DISCUSSION THREAD

**Discussion by NODRT** [TriliumNext/Trilium#4826] [General]
I'm loving Trilium so far except for the inability to search text in images. Is there a way to get OCR (Tesseract maybe?) as a plugin with Trilium? Or is there any plan to include OCR in the product?

I've been on the hunt for a good self-hosted notes app with OCR for forever and Trilium seems to be the best notes app, other than the lack of OCR. If that could be implemented it would be amazing.

**Comment by meichthys** [comment]
We're glad that you're enjoying Trilium. We do plan to at least investigate OCR, however, it has not been started yet. If you want to follow the progress on this, you should follow it on the new TriliumNext repository, since this repository is now in maintenance mode: https://github.com/TriliumNext/Notes/issues/115
